apps/notifications/env/.env.test

```
@@ -1,5 +1,5 @@
-RPC_NODE_ENDPOINT=https://consolerpc.akashnet.net
-API_NODE_ENDPOINT=https://consoleapi.akashnet.net
+RPC_NODE_ENDPOINT=https://rpc.akt.dev/rest
+API_NODE_ENDPOINT=https://rpc.akt.dev/rpc
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_NODE_ENDPOINT key should go before the RPC_NODE_ENDPOINT key

(UnorderedKey)